Tasks/DownloadBuildArtifactsV0/main.ts

```diff
@@ -71,7 +71,7 @@ async function main(): Promise<void> {
         var branchName: string =  tl.getInput("branchName", false);;
         var downloadPath: string = tl.getInput("downloadPath", true);
         var downloadType: string = tl.getInput("downloadType", true);
-        var tagFiltersInput = tl.getInput("tagFilters", false);
+        var tagFiltersInput: string = tl.getInput("tags", false);
```

Comment:
Is this case insensitive to the key name? Otherwise, this change will break task input that has already been saved.

Reply:
Yes it is case insensitive.